core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/io/ParserTests.kt

```diff
@@ -145,9 +145,9 @@ class ParserTests {
         parsed.toList() shouldBe listOf(1, 2, null, 3, null, null, 4.0, 5.0)
     }
 
-    @Test // This does not yet use fastDoubleParser!
+    @Test
     fun `converting String to Double in different locales`() {
-        val currentLocale = Locale.getDefault()
+        val systemLocale = Locale.getDefault()
```

Comment:
is it machine dependent test?

Reply:
no, this backs up the system locale and restores it at the end of the test. During the test we set `Locale.setDefault()` to be machine independant